docs/Protocol messaging.md

```diff
@@ -1,8 +1,7 @@
 # Protocol messaging
 
-All protocol messages MUST have have the following 2 properties:
+All protocol messages MUST have have the following property:
 
-* protocolVersion - describes the version of this protocol (e.g. 1.0.0)
 * messageType - describes the message type (e.g. Command)
 
 `Note`: When sending messages, each message MUST contain a `handle` numeric identifier. This is then used when responses are received from the device for matching the responses to the messages sent by the controller. The `handle` has no programmatic significance for the device.
```

Comment:
There are a couple of `Note` indicators in this section that seems needless, as these are already MUST normative clauses.

Could we loose the `Note` indicators?